### DISCUSSION TITLE
Cypress! i want to automate the login using GoogleAuth Provider

### DISCUSSION THREAD

**Discussion by arturoCrisanto** [community/community#123234] [Programming Help]
so here's my problem i already tried the one in the Documentation in [Cypress](https://docs.cypress.io/guides/end-to-end-testing/google-authentication#__docusaurus_skipToContent_fallback) but it didnt work for me, already set up my google token, but still encounter some error

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @arturoCrisanto ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Comment by CopperEagle** [comment]
Hey there @arturoCrisanto 

"*... but still encounter some error*" is too generic. If you provide the community a codesnippet and the trace of the error, you are more likely to get feedback.

Cheers
